packages/css/src/formElements/checkboxgroup/checkboxgroup.css

```diff
@@ -38,6 +38,11 @@
   flex-direction: column;
 }
 
+.md-checkboxgroup .md-checkboxgroup__options--grid {
+  display: grid;
+  grid-template-columns: repeat(2, minmax(max-content, 1fr));
```

Comment:
Could the grid possibly be configurable in some way? E.g by reading an item attribute with `attr()` then assigning 2 as a fallback? This would make the number of elements in the grid more configurable. Open for other solutions/suggestions as well. :)

Reply:
Haven't used attr() before, but since it's stated as an experimental feature for setting any other css property than content I tried to come up with another alternative. Pushed one now, that sets the number of columns based on an component prop, using the styles-prop and template literals.

Reply:
Good solution. Seems good to me! 🙂